ursa/graph/graph.py

```diff
@@ -248,11 +251,12 @@ def add_foreign_keys(self, transaction_id, values):
         """
         assert transaction_id >= self._transaction_id, \
             "Transactions arrived out of order."
-        assert type(values) is dict, \
-            "Foreign keys must be dicts: {destination_graph: key}"
 
         if transaction_id > self._transaction_id:
-            new_keys = copy.deepcopy(self.foreign_keys)
+            # new_keys = copy.deepcopy(self.foreign_keys)
```

Comment:
Could you replace this line with a comment or TODO for figuring out a more efficient way to copy? Above also